7_troubleshooting/01_common_issues.txt

```diff
@@ -0,0 +1,157 @@
+=== Some reports are in "No Answer"
```

Comment:
This will need to be changed in 3.0+ for the new name (is it "No report" I'm not sure?)